hack/tests/e2e-ansible.sh

```diff
@@ -59,37 +59,32 @@ test_operator() {
         exit 1
     fi
 
-    header_text "verify that metrics service was created"
-    if ! timeout 60s bash -c -- "until kubectl get service/memcached-operator-metrics > /dev/null 2>&1; do sleep 1; done";
-    then
-        error_text "FAIL: Failed to get metrics service"
-        operator_logs
-        exit 1
-    fi
-
-    header_text "verify that the metrics endpoint exists (Port 8383)"
-    if ! timeout 1m bash -c -- "until kubectl run --attach --rm --restart=Never test-metrics --image=$metrics_test_image -- curl -sfo /dev/null http://memcached-operator-metrics:8383/metrics; do sleep 1; done";
-    then
-        error_text "FAIL: Failed to verify that metrics endpoint exists"
-        operator_logs
-        exit 1
-    fi
-
-    header_text "verify that the metrics endpoint exists (Port 8686)"
-    if ! timeout 1m bash -c -- "until kubectl run --attach --rm --restart=Never test-metrics --image=$metrics_test_image -- curl -sfo /dev/null http://memcached-operator-metrics:8686/metrics; do sleep 1; done";
-    then
-        error_text "FAIL: Failed to verify that metrics endpoint exists"
-        operator_logs
-        exit 1
-    fi
+   # TODO @asmacdo to uncomment once new kb layout is merged.
+
+   # header_text "verify that metrics service was created"
+   # if ! timeout 60s bash -c -- "until kubectl get service/memcached-operator-metrics > /dev/null 2>&1; do sleep 1; done";
+   # then
+   #     error_text "FAIL: Failed to get metrics service"
+   #     operator_logs
+   #     exit 1
+   # fi
+
+   # header_text "verify that the metrics endpoint exists (Port 8383)"
```

Comment:
Add a TODO here to add a `--metrics-addr` flag to the ansible operator and default it to 8080.

See https://github.com/operator-framework/operator-sdk/pull/3440

Once we have an equivalent PR merged for ansible, we'll need to update this port to be 8080.